src/tyr/serializers.cc

```diff
@@ -34,6 +34,55 @@ namespace {
 midgard::PointLL to_ll(const LatLng& ll) {
   return midgard::PointLL{ll.lng(), ll.lat()};
 }
+using FormOfWay = valhalla::midgard::OpenLR::LocationReferencePoint::FormOfWay;
+
+FormOfWay road_class_to_fow(const valhalla::TripLeg::Edge& edge) {
```

Comment:
i moved the patched versions of these functions over to the serializer where they are used. ive also converted the tests to check them after serialization to get the full round trip whilest also avoiding polluting the public interface with what are implementation details.